tools/gen/23.3/rpk-registry-schema-get.adoc

```diff
@@ -8,7 +8,9 @@ potential (mutually exclusive) ways:
 

 * By version, returning a schema for a required subject and version


```

Comment:
What are these ^M characters?

Reply:
Not sure. But they were already present in the file. I'm guessing there's something to do with Windows formating as opposed to posix formatting.

Reply:
Yup. Difference in EOL/New Line formatts
https://stackoverflow.com/questions/1889559/make-git-diff-ignore-m